data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -22,34 +22,35 @@ class TokenSettings(ZkLendSpecificTokenSettings, TokenSettings):
     pass
 
 
-def convert_string_to_decimal_1e27(string_big_num):
+def format_reserve_data_number(string_large_num: str):
     """
-    Method to convert big string number to decimal.
+    Method to convert large string number to decimal.
     Example:
-    string_big_num = "800000000000000000000000000"
+    string_large_num = "800000000000000000000000000"
     return = 0.80
     """
-    decimal_big_num = decimal.Decimal(string_big_num)
-    scale_factor = decimal.Decimal("1e27")
-    bigResult = decimal_big_num / scale_factor
-    # Have to cast "0.800000000000000000000000000" to "800000000000000000000000000"
-    formatted_number = "{:.2f}".format(bigResult)
+    int_large_num = decimal.Decimal(string_large_num)
+    SCALE_FACTOR = decimal.Decimal("1e27")
+    # Example: cast 800000000000000000000000000 to 0.800000000000000000000000000
+    decimal_large_num = int_large_num / SCALE_FACTOR
+    # Example: convert 0.800000000000000000000000000 to 0.80
+    formatted_number = round(decimal_large_num, 2)
     return formatted_number
 
-def get_value_by_name(data_list, name):
+def get_value_by_name(data_list: list, name: str):
     """
     Method to return a specific value in a list based on the name.
     """
     return next((item['value'] for item in data_list if item['name'] == name), None)
 
-def create_and_fill_new_zklend_token_setting(reserve_data):
+def get_token_settings(reserve_data: list):
```

Comment:
Add also the type for the contents of the list. Applies also to dicts, sets, tuples, etc. 😉

Reply:
Fix it in this commit 73f712a53242c2402dbad84e335c8b84ae1e3ebf
I add typing for the list parameter -> def get_value_by_name(data_list: list[dict[str, str]], name: str):